### PR TITLE
MAINT Simplify warning in `_persist_input`

### PR DESCRIPTION
**Pull request by jjerphan**
Fixes https://github.com/joblib/joblib/issues/1370.

### CODE REVIEW

File: joblib/memory.py [hunk 1]
```diff
@@ -849,10 +849,7 @@ def _persist_input(self, duration, args, kwargs, this_duration_limit=0.5):
                           "(results will be correct in all cases). \n"
                           "The reason for this is probably some large input "
                           "arguments for a wrapped\n"
-                          " function (e.g. large strings).\n"
-                          "THIS IS A JOBLIB ISSUE. If you can, kindly provide "
-                          "the joblib's team with an\n"
-                          " example so that they can fix the problem."
+                          " function."
```
Comment (ogrisel) [inline joblib/memory.py:852]: The locations of `\n` signs make no sense to me. Let's remove them all from this message and let the terminal of the user wrap the warning message instead.

**Review by ogrisel — APPROVED** [review]
Thanks @jjerphan!

**Comment by jjerphan** [comment]
I am not sure if the CI failure are related to those changes. What do people think?

File: joblib/memory.py [hunk 2]
```diff
@@ -843,16 +843,13 @@ def _persist_input(self, duration, args, kwargs, this_duration_limit=0.5):
...
+                          "performance problems "
+                          "(results will be correct in all cases). "
                           "The reason for this is probably some large input "
-                          "arguments for a wrapped\n"
-                          " function (e.g. large strings).\n"
-                          "THIS IS A JOBLIB ISSUE. If you can, kindly provide "
-                          "the joblib's team with an\n"
-                          " example so that they can fix the problem."
+                          "arguments for a wrapped"
+                          " function."
```
Comment (ogrisel) [inline joblib/memory.py:852]:
```suggestion
                          "arguments for a wrapped function."
```

**Review by ogrisel — APPROVED** [review]

**Pull request merged**